src/api/initialDataTypes.js

```diff
@@ -257,6 +257,8 @@ export type InitialDataRealm = $ReadOnly<{|
   realm_uri: string,
   realm_video_chat_provider: number,
   realm_waiting_period_threshold: number,
+  // TODO(server-5.0): Added in feat. 160
+  realm_web_public_access_enabled: boolean,
```

Comment:
Hmm, I'm not finding this in the docs!
https://zulip.com/api/register-queue

Aha, it looks to be on a different endpoint:
https://zulip.com/api/get-server-settings

Is it also present in the /register response? That's what this type describes. See the comment at the top of the file, and the `InitialData` jsdoc it points to:
```js
/**
 * The initial data snapshot sent on `/register`.
 *
 * See API docs:
 *   https://zulip.com/api/register-queue#response
…
```

Reply:
> Is it also present in the /register response?

It looks like`is_web_public` is present in https://zulip.com/api/register-queue#response but under the `subscriptions` key, which looks like it should already be present:

https://github.com/Fingel/zulip-mobile/blob/0684547f32fa00b4380a1158308e821a72edb5bd/src/api/initialDataTypes.js#L359-L361

Since `Stream` already has the `is_web_public` property.